<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prodcat</class>
 <widget class="QMainWindow" name="prodcat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>771</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DecoType Naskh Special</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion de Produits</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_prix">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>340</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid gold;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>370</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Code Barre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>340</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_codebarre">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>370</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid gold;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>370</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Categorie</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>340</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>340</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Cycle de vie</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Nova</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.75, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);
border:1px solid gold;</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cyclevie">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>340</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid gold;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idprod">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>340</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid gold;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_categorie">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>370</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid gold;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nomprod">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>370</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid gold;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>370</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>-40</x>
        <y>0</y>
        <width>821</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.499807, cy:0.489, radius:0.75, fx:0.499, fy:0.488909, stop:0.0795455 rgba(40, 109, 189,255), stop:1 rgba(30, 30, 30, 255));
border: 1px solid black;
border-radius: 15px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../light-blue-gradient-ui-gradient.jpg</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_recherche1">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HP Simplified Light</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.75, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);
border:1px solid gold;</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rechercheprod">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>280</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Corbel Light</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid gold;</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Rechercher par ID</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_trier1">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HP Simplified Light</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.75, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);
border:1px solid gold;</string>
      </property>
      <property name="text">
       <string>Trier par ID</string>
      </property>
     </widget>
     <widget class="QTableView" name="tabproduit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>621</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.499807, cy:0.489, radius:2.5, fx:0.499, fy:0.488909, stop:0.0795455 rgba(255, 255, 255,255), stop:0.6 rgba(100, 130, 255, 255));
border: 1px solid gold;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier1">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Nova</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid gold;
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.75, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supp1">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>20</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Nova</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid gold;background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.75, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_suppnom">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid gold;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <zorder>label_28</zorder>
     <zorder>lineEdit_rechercheprod</zorder>
     <zorder>pushButton_recherche1</zorder>
     <zorder>tabproduit</zorder>
     <zorder>pushButton_trier1</zorder>
     <zorder>lineEdit_prix</zorder>
     <zorder>label_11</zorder>
     <zorder>label_3</zorder>
     <zorder>lineEdit_codebarre</zorder>
     <zorder>label_4</zorder>
     <zorder>label</zorder>
     <zorder>label_5</zorder>
     <zorder>pushButton_ajouter1</zorder>
     <zorder>lineEdit_cyclevie</zorder>
     <zorder>lineEdit_idprod</zorder>
     <zorder>lineEdit_categorie</zorder>
     <zorder>lineEdit_nomprod</zorder>
     <zorder>label_2</zorder>
     <zorder>pushButton_modifier1</zorder>
     <zorder>pushButton_supp1</zorder>
     <zorder>lineEdit_suppnom</zorder>
     <zorder>label_6</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion de Categories</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_recherche2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HP Simplified Light</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.75, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);
border:1px solid gold;</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_recherchecat">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>280</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Corbel Light</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid gold;</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Rechercher par ID</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTableView" name="tabcat">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>621</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.499807, cy:0.489, radius:2.5, fx:0.499, fy:0.488909, stop:0.0795455 rgba(255, 255, 255,255), stop:0.6 rgba(100, 130, 255, 255));
border:1px solid gold;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_trier2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HP Simplified Light</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.75, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);
border:1px solid gold;</string>
      </property>
      <property name="text">
       <string>Trier par ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idcat">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid gold;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nomcat">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>360</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid gold;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_41">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Nova</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.75, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);
border:1px solid gold;</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>280</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Nova</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.75, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);
border:1px solid gold;</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supp2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Nova</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid gold;background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.75, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>0</y>
        <width>821</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qradialgradient(spread:pad, cx:0.499807, cy:0.489, radius:0.75, fx:0.499, fy:0.488909, stop:0.0795455 rgba(40, 109, 189,255), stop:1 rgba(30, 30, 30, 255));
border: 1px solid black;
border-radius: 15px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../light-blue-gradient-ui-gradient.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_suppcat">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid gold;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_43">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <zorder>label_29</zorder>
     <zorder>pushButton_recherche2</zorder>
     <zorder>lineEdit_recherchecat</zorder>
     <zorder>tabcat</zorder>
     <zorder>pushButton_trier2</zorder>
     <zorder>lineEdit_idcat</zorder>
     <zorder>lineEdit_nomcat</zorder>
     <zorder>label_40</zorder>
     <zorder>label_41</zorder>
     <zorder>pushButton_ajouter2</zorder>
     <zorder>pushButton_modifier2</zorder>
     <zorder>pushButton_supp2</zorder>
     <zorder>lineEdit_suppcat</zorder>
     <zorder>label_43</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-50</y>
      <width>1200</width>
      <height>721</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../gradients-for-photoshop-background-ui-crimson-tide.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>0</y>
      <width>931</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.499807, cy:0.489, radius:0.75, fx:0.499, fy:0.488909, stop:0.0795455 rgba(40, 109, 189,255), stop:1 rgba(30, 30, 30, 255));
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>label_21</zorder>
   <zorder>label_42</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
